### DISCUSSION TITLE
won't go full screen, taskbar still appears

### DISCUSSION THREAD

**Discussion by Megumiiiiii** [ppy/osu#24819] [Q&A]
### Type

Other

### Bug description

won't go full screen, taskbar still appears

### Screenshots or videos

![Screenshot_92](https://github.com/ppy/osu/assets/98658943/21ddf125-9863-4372-ae65-9f4f846e9084)
![Screenshot_93](https://github.com/ppy/osu/assets/98658943/014347a3-7d97-4608-a773-1cd3f6e29e3d)

### Version

2023-914.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/12614362/database.log)
[input.log](https://github.com/ppy/osu/files/12614363/input.log)
[network.log](https://github.com/ppy/osu/files/12614364/network.log)
[performance.log](https://github.com/ppy/osu/files/12614365/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12614366/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12614367/updater.log)

**Comment by peppy** [comment]
is this a custom taskbar? it's missing the system tray for instance.

**Reply by Megumiiiiii** [reply]
yes, RoundedTB. But now it's normal, I deleted Lazer then reinstalled it
